tests/unit/helpers/orientation-validator.test.js

```diff
@@ -0,0 +1,17 @@
+import orientationValidator from '../../../source/helpers/orientation-validator';
+import {ORIENTATION_ALL, ORIENTATION_LANDSCAPE, ORIENTATION_PORTRAIT} from '../../../source/constants';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)